lib/index.js

```diff
@@ -12,19 +12,20 @@ const githubRunNumber = process.env.GITHUB_RUN_NUMBER;
 const githubSHA = process.env.GITHUB_SHA;
 
 async function getCommitData() {
+  const prettyArg = `--pretty=format:${'%H %an %s'.split(' ').join('%n')}`;
   const log = await git()
```

Comment:
btw `simple-git-promise` has been deprecated. I think we should use straight `@actions/exec` here as well